### ISSUE TITLE
Passing a different $length to LimitSubjectLength

### ISSUE THREAD

**Issue by benr77** [CaptainHookPhp/captainhook#85]
I want to use the LimitSubjectLength rule but with a 60 character limit instead of the default 50. How can I pass this value to the rule?

```
    "commit-msg": {
        "enabled": true,
        "actions": [
            {
              "action": "\\CaptainHook\\App\\Hook\\Message\\Action\\Rules",
              "options": [
                "\\CaptainHook\\App\\Hook\\Message\\Rule\\MsgNotEmpty",
                "\\CaptainHook\\App\\Hook\\Message\\Rule\\LimitSubjectLength",
                "\\CaptainHook\\App\\Hook\\Message\\Rule\\LimitBodyLineLength"
              ]
            }
        ]
    },
```

Thanks

**Comment by sebastianfeldmann** [comment]
Sadly this is currently not possible, I think I can add this functionality by allowing something like the following

```
"commit-msg": {
        "enabled": true,
        "actions": [
            {
              "action": "\\CaptainHook\\App\\Hook\\Message\\Action\\Rules",
              "options": [
                ["\\CaptainHook\\App\\Hook\\Message\\Rule\\ LimitSubjectLength", [50]]
              ]
            }
        ]
    },
```

Then check if the config entry is an array and extract class and constructor arguments accordingly.

```php
foreach ($rules as $rule) {
    if (is_string($rule)) {
        $book->addRule($this->createRule($rule));
        continue;
    }
    $book->addRule($this->extractRuleFromConfig($rule);
}
```

**Comment by benr77** [comment]
Yes - it's certainly a valid use-case to be able to pass parameters to the rules. Thanks!

**Comment by sebastianfeldmann** [comment]
I just released version 3.5.1 with the possibility to use the above described configuration format.

```
    "commit-msg": {
        "enabled": true,
        "actions": [
            {
              "action": "\\CaptainHook\\App\\Hook\\Message\\Action\\Rules",
              "options": [
                "\\CaptainHook\\App\\Hook\\Message\\Rule\\MsgNotEmpty",
                ["\\CaptainHook\\App\\Hook\\Message\\Rule\\LimitSubjectLength", [50]],
                "\\CaptainHook\\App\\Hook\\Message\\Rule\\LimitBodyLineLength"
              ]
            }
        ]
    },
```

Thanks for pointing out this missing option :)

**Comment by benr77** [comment]
Brilliant - thank you for such a quick fix too!
